### DISCUSSION TITLE
Server 2022 support.

### DISCUSSION THREAD

**Discussion by birdman3131** [gameserverapp/Platform#202] [Suggestions]
Server 2019 is approaching end of mainstream support is about 18 months. (Jan 2024) While server 2019 is still perfectly fine I would personally not be the biggest fan of purchasing a copy for a new install when I could purchase server 2022 for about the same price.

If need be I could fire up a server or 2 with 22 on them and do what testing I can from my end.

**Comment by Leonrrcs** [comment]
Where did U buy it dir the Sams price? 😵‍💫
I only find prices 150+€ 😅
